app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/background"
+        android:layout_width="fill_parent"
+        android:layout_height="180dp"
```

Comment:
As I told you about colors, there is also a `dimens.xml` that will contain all your dimens resources, so it will not be here.
More information: https://developer.android.com/guide/topics/resources/more-resources#Dimension.
Please change it everywhere.